server/model/monitor.js

```diff
@@ -1002,6 +1048,7 @@ class Monitor extends BeanModel {
             monitorID
         ]);
     }
+
```

Comment:
why?

Reply:
Hi @Saibamen, I have remove the new space line. The IDE added it. Can we have a pre-commit hook to ensure all code format is consistent in each PR?

Reply:
@louislam We need to add more rules to ESLint or add [Prettier](https://prettier.io/)

Reply:
@louislam: [Prettier Example for `monitor.js`](https://prettier.io/playground/#N4Igxg9gdgLgprEAuc0DOMAEALGMAOamAvJgE5wCOArgJYUAUAOiLgWiwJQDcTUkUDJgAmAQwCeAKyKkKNenGYgxUjiB58BQ6jDAlyVOoxYrpAenwAbagHNaUMzrBdeUS3CwxaAWzgAvaDh9OSNFEwlzK1t7My9fAKg4Fz5TNAA6OAAPeChhBicNXIj0rJy8uP9Awq0sUUzaCBkDeWMQOoa1avQsYEwABTIIXxhsOGoiAF9gwwUlNLSLQeHR8eT+bsxeywgbABpMAFU+-YARAHkAdQA5fb6AUSuTgEkrgHF9gDNLWnwAZRhRDBxvsACo+OAAGR2NjgZEwU1kM1a8zMKLQZDAji8ljWNU2mF0+Hse0wRKgJOEggASnA0BBLAA3OD7MCjMAAawAwrCvB9aGBAczMKy4Bz-oDxpyIMIhTCYCCIADLJzvggYE8oFSIEN9mgPF5yftvGg0JRLABFaiw8T7fCNGA2ChoS3Wo2UPAAQTQ4n4uo8v31xP2bHwVxglm8+0d+DALrI4nh0xaYRAKJ0tEsAFo9WQmWRcRtelTE4jk0oKMIAEZwURQTNQaVJdSuPG9ABCNagAFlG5YS81QuW4FXO-XG2ZhLQMGZq7XM95ewXBD1MFdFbQ+QKvNB+yFZiwUQ3efzAQ0oEuhL0BhBMgmEQP96mFvhBreLyuZQuezLd0iUyiBD5Gx30wPM0DPJNBwPBYUXwUQOVEGE0mkaAuDSMCzxbDZRCJAURUgx8UQXYRqHcNAzBwk8RRA3oDnwCoAGlqG8UQA1zWFfzLaDHHo8FM3ZZjRGzWE8xozBOXg0ZRErdwTkEAAJPB8A9GFYE4qCnzMPCpJkuBM0pNBMxDTNELVNY+DMAAqSy+EwSzMAwCU0CQWz7Mwdz3IABn0c5rlcjyPIARn0I5-ICzAACZ9HuR4XleVyzE0SxRBNTAeygWgYAgOFSgQYQiA7WtvzgPtgD4WyPKsmyoHC+yaSBMgatrTAIErSRRU8bBAQMURhATLLSVEMg9QJCBMAAKV+M4rkwD5stJagZP5CqPPss43ATNBsAgAB3TBEjAWk0CGhMxABUaFqW5waoC+yAAEKAawRNjONqOomFb3MsxKbvclKfT0LK+kW74wEm6aGC23aQUQpoPlESw9U4TZPoC9wsFayR9DK37woC2hhCQAlsCnNICd2VG8f20RfCJkZSagGnmUpvG9VyA4yEsOmSfSNnhA5ywKdxgKJlcFnMA3TAGHp3m8oF5Gcap8LMbSahOf0GXVc51wlY+89hY8yXIe2naYZsNAFfFgKVYBc39FEHbREy4nSblM20AYQpdfFx61ZqzGdY8vXPqq1G6o8P3MGazGOuJ7qKF6-qxrg4aggG8GrjDzAHojxqiGAV72rAGA9dqn6Av+-hRozhh7DAawZQDQRMtoJkTkBUQNbIK1Lf1pX0f29dN1PaAnhOCEpywUhgFF8qDfcgfviEUgHadrAqTSPlciUBcMqysgAH0jw3E9t3PEB9hYTBd8y7KD4J-QAH5MBYfYAG0rY8zXyfFgBdQpxZzThAwAes5-YfEwEvGAvclYBWPsPM+Y8J4YDfmAtI8DT5nnvsIX+XcrSBzxsHeewoNi2yaKvZ2ms3aw09mLYhA8zqd2np-dyBNuakx-sQuBTN2HpEZr4IWsCPJqy5i7dIIjBFCOvh4bahMxFpGWNKSRQjtoYH4XAXhaRVEwHUco2BdoyAwE0QYmAeilYsUyI9MgtBaSaIsVYmxaAzFUx2nAWgNhcCaNce43Azi8bwS8EyTRATW7My4V-cQ+ANHyJgJEsJUiJawBEgjTR9h4C5gRn48KVjxAanSQyFJ8icl5OSYLFhmB2RwHEDtbKcjNaVOqbUrJAUshEnjGuY8W4zypLQHcRmulhB3EyG08QHST5dOgJ7ZphsbANgoCCRGmi5RPFmdlOACyPacGme5aghACZwBOLtKAPS6LgRlIcnaUApnlPscOBQxdnLyNuZOCgDztlRzAIdeiw4D6OSBGgSAMpHlUI8B6T5cBvnCHFP8wFtJrnhPcgZA+Tp6RMgPrEqJmikUosZHAdFcT3nYtpKivFOY8xYsEMi4luLfkiVhISylyUMBUrQKRIx8ikVMpgCytl7z8DjGwAqSpxz5H8q2kKhAhKIAclhAfAQAJ7CwixdKyph95VO0SGQKVMrD7aOVTqg+2i+WvlyXUnmaQXw3nENg95GCJlQCQZPd5ZC6aw3ed4d0MADg5nUXYz13rYS6JuZ6voKU0A1LIGa0mHq8ChpNBG4Q7rPUKlwn6vAKb+RJrwL8ag4KTRdiOqZNNMAc15rQAWk0plCUd0rClOAUooAHTPv8ax5IsU1rrQ2ptZ4W1BnKYw2teo4ziHbQCQdcBh3vNEDobABaRjSmCTOudsip0zouNldkfzunyOnSMddZBN0AjPqukYhyWL2EXaeoYGr3nRjAALJZZAYwC1vU+sA14sqVmoB8R9MYP2tW-a+mMy6F3yLvSBxN5S71sQZPyOAVweFgbfTBuDCGBFQbfX06S7gNlLI8K8TD-ScOI3hbrOhSsjZ1wbnAJu4FAkHI7jAqRjDsblICvMNIjD3kBSkjKYamjeOwicWxjylZpQjvkWJvq3GPJ3rbOJ39YB5PSZE+5cDHheod0U3OzTAIZPuUHfyA+u7sAH3GEqyTKUjMmbMzmfTmBDNgGMzOg+cETSaMc85kYrmw32bFYKiAwrjEColVAd5s9iFEKVr7RqIgO4EPhHPAKodhb2VeB4KOlg+xkKjvgKwNjhAXRltfaAt98ypezjF56wBrzeCnHAAAPFCRocBC4dQa7WcQAA+N+v8uul1uuXDylc9DUPNp7FGxCqtR0ds7DecoPRZaUL8O4EI7ichBNfGAaRLL7Ftugpme3EJpEgNsOEAAxKkZwuwlb3nfW2W2JpnBeASRCmBppbbSLbbBGtjsPwuPJO4VI7ifZvvvH7pBH6v0wG-eRD9-4Jai5VayWc+mwswOZuEtZCtufDbUi6bY60ADYAAsmAECAuJFnIBmB5IghBH0F+IBHNXxM-sFKpIOJUnO5yTMAB2IngU+dZxzk9fOGBW02AG6tIb7kKeNkJ3qUn+Q7ODRNExvG022zfo+LCTeSwVccQANRM5ciATAJvcecC+xAXt5IlDjtJ2sEWSXkfVVqpgJ4RARhBBBBCX45Phn0ATHakeNUEDYeHFDirou-b5zE-STs0uvqy4lr0ojw4hkjLGQgs8E3FZUy19qdwtZpbmtacHnPmDoBeyDq79yKWPehtThdBPJe+63Uq7narbek9h1T8s1Z8ySMa+yd3zA8nE+l+-kP9ZiNa-uSRw3lHFXm8jQGr32sIuqvx+L331LqepynP2Rcq5o+ApF6n1czWuyzkHKOQvxLHfl-u872vtOY1N-P6+l3sXmwv-J52QD74aEaR4bL54+zj6T7t5l6uygG6QbKP5L6YCN6d7pa1DgqQoORHrjAkJArb7d75xtbFyAHfSfQLaYHwBQo4EAqNibKTbRbj4ZwWpDR6iwHpDwRfJUG-I0GwpoAHwoRQBIH14oEr4e7iiGK3ZlYi4pw0ybAwawhTANCYC-AqoZZkocT2COT8BJAH6fSOSGK1wQDn4eQDwvhwCwYQDjCFRTz7SkSWAJZowZYOK0j6CeTkZUxmFLAyJjBNCJB7S1Y+HjDsEALEJ4izi2EjZSzIzEBdaTblKgI1jqhJIZJ9ikDfwpEFIOEiEUYQIMAACEDmSRJSqRJhQiERJRWR+ggUjhVMyBuRUsn4EAxUZRsCRsFRmRCMmADWkUnkrRCSWgieaQ2wNgDuSRiS+SXRWUY02wO0+wToGWA0EUnknQtRCSBmxRnRaRvRaxsCgBdRORVMZgZgmAWeLWOAsIQQNOlqywvhGO+AZ0uhCSxxGOuQcAW8w4EsECR4OASkHACKkCixiMGoc0+g1AbxHxwgHhQiRs+R5hlh1h4xAAPkiTEnEiQMQKQCwP5lwAwese5PCQ0Iid1CvLNuvJvPYMIOtCmFJIYhEdDlfGDnfA-JDu9lSCcEDhPgAJoEjgiYAcm-CcjQ4fwAlKzfyQaikeQI7lL1FUx4hTjnb0AYA2H6BwkUAIloA2HQmwKJHNSkAbyThoBRKCA0k1h0lJHO5SJoJMmHwslw5QnlJoIVD6AbxTgQDtzwBgi+BdgZhLwMCpCqy6CeyP5KxoJ-K4GkC+RXDakNHsFkxoDH7nJHLBl4n4lhk0H6BfA-DQrBHplOQhkHHf5KwvEnBqxh7lKwkKlKkwA2H9FUxoIkRkBh76CpAMBOngjW6TgfAfD+nFAMCElWGalJFfYdmXwgB6gCDCCWlCJTAlQjQF5pmdicZllnxuG7Eu5FlUwwDxipnrFGyawYpBCYlYmsBKRXwololRIYknkNIJq4kLn4mVQnGHKDxYDCBjSdaYA0CwiOJmAkweKQIQC9TEguzkhEDVhAJpyjBM5oJkjAQgCqbhQDwGHyh8mkCtnW5ZFWhnA9mhGPnhQvF04M4OZWZ6AmaIVOFYCOYegzryQ1h8bYwRb4X4x5Gaw2aKKFbHkwWkX3kUX1mkU0UjB0W9QcTMKSn4ksCCXbTWJ+DlnIBM6K78hM4W7yLy4yiK5wDK6ayeY2aY57bmo6UuZW72ZkZ8WL6HH4l4ghhoAqRqhnC8ToCsbiWwIWISQihQpHRniPKeT7AvEBriTjBZTeBRyqSeBjSGmR4OSeU7gUDmZSzWVIDHF2AjCLQnZDBmANgyjmCZVwBmBTisq0hmAADMAAnCTn0WZWPkXF6ozDOtlLQH4MOETPkSCuqLPuAVspVRFpVSMZxnAF+qMSwDadDgAAZvwAAkwAms6iEwuCAwEKQ0QQ9lZ8RANO7QjQI1eFzF7keIEADlz0Yl21eMIimiEilVAUHFRMcZHFmAF5LAcoaEWUUIriZAEkbBnVzlSsHGcZUmCYz8vQjCRMzBKcbB2l4myMUwRMM8H1R14UFQVh7KGRkxfY9kgUnk6NdkmAnkaQAAHCZVTIJvxrubDQFJJZQSwETCwPANkH+TAN4ILDhPlvamYJkLgPTUbpkPTbsIzaDGHizfTdwJQMQNjSVbsD4KZBRLBh8GLSxDCGYK4pWPgDLRLThIaN9JZILcLbjdzXlrzWfGYOBLMsOJmFkKyLWDCNwAyMQJWEVZrSLa-OdVTCwAGmQJmLZbABTUznRBUJmExCxIlObibhhNAPjUIt9ZrITUQM-MDawYoBHfRUJsjFDRMDDSTXjN9dRbRQnWQKnWnRMKHeFBYjSC8h1MCuas8vcjAMJp9VTFkQTIKDmY8pDDQTEXEQ+Wne5NNiKGKDQVKDKM3U5Ppa7KCpQcOI3XwcGeuY+fnV1TGY+fueapareNgnWZZRsEvQmKSWvJgBvNsL1EoBvdDprBvSvXPdtUbBvZgAAGRX2kgmppAhJMir3MWti-EEDu2mJv2EAf39jXi3gnYJzwAf0ewb37Dt0d08Z-Ef3LVeVEzWXQP7XV0QMixbXIMtSIMWomqZkIx6hT3bV7UrVaJKQ-2kAhgf14PMUENeVEPsAkNf02WhUUMJL7H4myn4mwlUPoA0Pf2hXP0JKcOCDcMMNqj6D+H0NpAf0MDwOhUwPoAFmsMWXrG9UygDU7ylb7yjUTVTXmozW4Iej1CNDvaINEyTXMES7EgbjiAMACMWwTCbVMOeE7B9WqNDXqPZSaOTXTVMyzWYD6MdA72GC0gwD2OKMJIDxOgON4wL2kzsUyLSjXlM6wARi8U114w2NCMf1pCVIQqLahJ4JwBn3MVOj2xklf1hgRjWOINgOO3HU+qIbaWkVeamZ6U1PhS44JoeaNM2a44F1KzvnnoipsUzpno3qtMBQRqHph6XrYD7qTOrnPxDN7obpbo7hEzgkyiQmtMp2FPMPk6IxBDgPbXFNb3OzrXpAhBBOVMrXyPrFsPrHWloA2D6AjWTVOhpDhloBTCZiYCvO0jvM0EgilB2ORPhSwUgXoV9mYUIzYW4WYDfMoVekFOhNSIvGchsjsjCg8jjKChfH0Np4Y56hTmpOAlYCHSGIgm24AiGKIstmQvoTQuta4UgssVSytUCyeyPyYOKjSr0gJMsAJUpMQPKP9W2BqN3b5gXyYBjWeM6PeO4JouigYtkvBM3P4XbkJiHMk0DzhhoAUvEG2Hd1chYsIKKBOiqtHU6sUslPb034PGCgbIUvSzAlQBzT6uoPIOwmtUrJzJz70E31w7p6R6DJB7tJDzV5n58NHXCsuMgDDWSvSvaMMxyvCgIx9h8zciGJV72ohPEsJIUKdSkzps8hZth5Ou6susQBuvMtSJ3P4VTBbishSxwCRvsN5FwAKKFowiYD5GYlM5riYB+4B7KvYvwCJIGBGnoBNgtvMW9WwiDBkBivSGSssASS2C4Dk5kDzvTloOmFONzvZSLsaOSvtu+CVowjmvbUsPMVXvMPIswl5GWqHQmgZBQAMhpAghPBdirZnCvCvCclcUsCBSCsd3RuiuuPisMkgAZtYBWvDoDvgiU2B2NF0tYWMsTbfPKsUsSGoW+DIwm5DWrFdV3ttEPuDBPslCvtpBHAftfsHwMQHBdgegHwQg-sHzA6-B9DTQrYHxthnAnBckHxdjTRPAghnBUgHxjwJPinTv9xOP2BzSHvuOStvOMIXt7HEexkHnokAenkEDAdp15n-IhR9DVuL57PzkadSIMIdzTDpCMI7OPmovQB5hhXYGS6zTzQxzFwUSbsSCtP7lxIQAQIsY9snnmPkj6fIMsakBmPbkWMfBWOqemfhS1vz15GMJkz8DUYez1JVIJqcAydWlLnGhPNG4nn7C3n45TgefrMsDJehlLkfPGf1dBzmcHNjMeTRdxYAhpAUBWDwSKBmANZvwAB6XWv8lkXWj8SJH85UTAZAv8SJTATAaARuZgNgkYyl27EDRsGX7g5IIwmAcRAArH0cTTu11xl6ypWOF6MT5ZgCTnznh0zhxnVx1+Zbm1udgIMHtGI3cJuwew86V0zvsF+lgJV5Gviz8fYDDlfCbixvhyAL-Nt0dalzOZZ2Z3OUENE+kIeXyyACYpF-hWC+SNa5QmAHBXGdoroqKtlNAsl0D-oCwG98S4ZxGYcCZxj4mFj7i3GXjzp3BUT4+ST08yc1gJTxHfaOompw17WAoo80zwhSAAz41xmaQEcCC7Ofs7z1p1eTpwZEL-ichVSzh0eXFqoJ7PS9YGhzL54RlgZBWsdF2yeSzz1fb9SK4WLyIB73SLilT1L4dhypSjimihoVqrT4YkPXZ8H9Smioebb3jCL7Sxb1C9bzheh9gUNKbw5-e6y+akSr73H9p725JeeaiZrAXySvinryXyAB6PXx6GX0H-wSH3ivz7XyCAABogiG-MUO+dtBBldM40iQCRqPIs8QP99nuD-oU+-IQQD2BKC3VM4J8pdtc6-58x+F9t-F8nmchXAehftN8V9b9V-t8nl9AghUi9-4VT9O9m8GQ0hoBvyeS-zJda8jQ4+can80rn9M6cj18b+j5O-qZBbI+8X+v8eMgVXf7r8v+lfX-rvyZxdhO+QA-Eo-z+ZAI7gkkBgK8nxyxFzuadEAV2yH4jV5IAfWmD82AC4DI0GQTIGbXJBwAvm-QaxPVViQmMqBooWpJgwaDWJYkUwVEiNRa6L5V+fTQQI71AGz9y0A-d5qDEUD3dMwEUEQa1x55wCf+RfGvieSuC-BUB6xIgTPyZwKEiaE-DunoItySCn+8-RfueRX4wCVBrFTfi31j478NBBgs4I33UAEC++YggfmYKlZaDMwaGaJJNXQF8I0AM1ZfmQIwAsQMAFmYIXP20TRD0k-AlQj+UKRxC0AFgnMLQARjJCaQHwJ0NgHYEhDeu7xAobkI8Dxgihc-HJMkKzwKAqhGQv5hXgoDJCfSUAenBCAaEWC6ssAcMMCy6qwD7BpMeAeoKPK19fgVIAAGo6CEkxQzAdgOoGcU2673b3lIOn6+CRqgQ9gYsIOy+BkhfQOntsM4E0CTE+wlgbwIky-NR+wgbgawPEDJCLgbiDxOyiuFcDvEzw5IYIK2ZKC8YpgyQeIKQhoBZBDAeQYoPf5c8WWcZAyBCBSgwAn+bKbtr2z0GFc8YG8LIKKCUBHATgHoEECDhGo2kRqKhO4Jtk5SwieUlgWws-ABxA4QcdpZ+MKRWF6DemX3DhESwgbSliWaPWBIz3+ED8VecvJrur057OUP+2PIYbj0QHYkBU95PGC8RBBxIiYwMLaD1ScYqMwOsbNxhK32AJsvGew+VuixAo2ko4WAYIXSyAQsQYASgLktaK5KZguwXYTMCcBOC055ISAbwN4CQDPtfg3orgHYx+EkJlwDmHXLCBpakA0a6NbgCgROKBRwKwYuEANE4JDBiIOLeuCqhEAbhdcFAHQv8UqoX11SRJIcoCBRHhQ8QxoX4HXEhCwiVSELVQIGTACW9UO6fZGN8wDJOB+y+YwcjYRHK4creMLWhBCMcY2BnG6ouNtqK0a6jGBuCMsRWJhHKlxipASatOJ0KziaySRP0Tn3WIvEngECH3HfQsIFiii3UR2EQHfKXIWocIY0pOFJ6uICWUFIIBQBsCkQhorTF4h0WRpmBikWxMdruL1Cm8Eas0NWD7jhDVhZi-nUjvuM7HDkmuoXPPhwgTJ7IkylyCbM-CjKYAiYRwZGBeSXGHQVxKpOIm+NKSY1wx3kE3F+m7IhiOyng2GiMB+77Q4Ae0f7vOyUD9taSMACIt+OgrOkdolJXaCj0vbr9NWsNbVuCH-GkACJqRIiejW8jfNsJlYucYCCEHhR9yIknQN0SxrFiEk8NVSWJLjGItFJIsASSsIChaSsAQ-MiZmL0krCb2R1F4v20SCfEBoWhHkA5FzTkcPgpEC4ln3YnXEBUBIAlA5DGh5QLiFAFYS4SaDuEVhoHQahqIg7xtxxsrPUbyV8CiTKBJk9cSsPjpeTNiyNfQL+MRYI02ySRPbCpPp6hTu8ik6yTOUMmfd3INE3aHRIYkA8F2LAFiWaTYnjEYeu4riTxJ2h8TbmXPUURv1Jh-8WAMQmmDMLlIbAxp3gD0ESAFiK8EqxxSiP8xrDeA7QL1YcGlW8BmAngrwJmIYLQAoZDoZgdAoYOQQwAzADIQKAHWS54hppHoPoE8AYhVIyeWAX8QaGin3THpz08QH1IGIbA+QFI0SlK16jCB8wy3JgDK1JjU9vGSAKGekFOFfDNyrbKWPkS+lPSqkGkr+N93ql-cmpy2eAHIQelPBMAP018jV1yB-Sa2A4heM4U94zZt6ZzNIHKGbqrTZptAF9FRMfImTUkX41GlJMxrY08arTSOlDRWFk0uCntFgOrQdo1TwoztHMG7VCpSyQA3tPiH7VEAB0VKwdMLFs2ZHuRpGaoImG5R0iyQFIxDUKszI8CKRaGoVBgIJKOquVJIY9aKoICJj3cCKJxfypyECpDAQqIjAaBFV0hRUTQEEWKiNCkZ-FEq63TKNgFSqQBtpOVbKuOHypWhyIpVcqpwDKnVUDgtVedDJSardsvW7VEfPrPhC50SaZDUKsbOdmR45IaAG2cpEtlyhG5kjB2dtSdnuUAwoc9AO7LLkp0y5RdO5K8irp2I6gFYSukgw7p11HijdK6h81bpcyjm4+Q1o3T7qKAPmUfK2TADBRcEx6vBOgpPT1mtNZCxoMWXLICiAz0kRMK+fShWENIiY6Mn6f3K6rut8KRsFTh3BKETsTS19W+p-J65Og7QJpd5nSmGh-zx2nGL+UAsnagL2Iw0YYggBsCHc4iZ3dufc1V5ORmurTHkZAoy4wKQFYfZ-q-12FItkZzFdVkvOYpJ8veEvc1DDNw4VSU2ugbAFLAPgKwn8sNQaegs0k4zfu9E04vjJYCGDyZc0dZi1Bqj-BVpVM9HiKMGGwSJRLgkwGoQlbFiopDAHUQlMnHMCFqFAIxitQ85wg-GG1N+Y+TxDvkdUkQyIqU13pAU8gyig1NoiPr58VFhqe0KYrXqBibGTlCBidSlZaRoACqTVORHhmcZXF6qRVGQAmBmBBCI1MuaLKoX4UJZEKGACrJlkXxxZqsxWR-RVlqzfAvtQSFrKDpCYzw-c+JVA2rkNTxGbclYZ3NGAeUe5bsrGvsE9mHARoPsjAH7NMhqRA5U4SKnqEaUGA4qkc9gNHOSpxzKwW0jKo2GTkyg8qJoNOcVTKoVUL5HkCgDnLznSUGqhcwosXJ9YdV+5FcuthuNjIWLVUVitIAfDOUhjJR45NQqko8E8KpE6TOkDKhgChpDu6FFRRcquUqL24cAXeLYO167dvl9oS5dcrIDyilFIAQkBNKETpNx0BwKkBCBbKgqMA4Kv5aIABXQBwR5CsJqqJFajFNFSbRKcYqIA0hvyGAHNkJLpnkJSmTMi5hgCuZeV-RH8v5hl2hTtsqQ4JDKOSCxkbEBRavDnopNwXM9leRHORXYIUVfZblMaB5cWNwX5tr4nqL0ADH95qJA+msOVRmjABby5VpacjgCKFBPKhEJiYxHTzLmY5fUTyf1HU3Qxyz2mtSYtHGjxyRoy5aSUpLzORoF1tmlVNnk0A16IVBpX-EaeOTNBh84VSFdQib1DHm9pADYhlk2JOVUwlVxoU0BaCtDxgoRHaPUF2g6g9o4uhoDlDmonSZrxAHijBXLxK6K9jB21f1dgtZ5Lk4KyfeNanz7HNjM+1LcEJr3kV89bld6fiHl1qSRrKKna03i2s2S9ibet0jYN4sOod070D6JDM+k5hly70-6MiR6VECKYN1gGVptBhEioZ6m5qA9bmCPX2qIGd6LDAgUWTLqwA164jFPLTrqZ50UadIC+pXT7q30ymCTFQm-XiYy5EPN9VkyHWRpZ6lVPEAQpGhe870w6ZlXI35FQALU4LONZOsbGwt4WMa7tSqKHFqjopNpM3NqJsBUg+gnIcdsAr1DsDYukuSxjgNpAUbm26U4luE3o2Tst1tnBjVAoBDJc2VP8vUFusQUHdWFJ3NBa0yg0MZzoiIPje22u63cQR+wR7s92ghpBa117XMXkXE1pUfwMEwKPysPFIbBR-JS4NGRwXFcFepAEaoxPmgw8+YmAYjaRsoGabYUUwX5tJoyBNSjV-QzkdVJ24abWNICjLllDtyjFrcVGagECjjJAaCuiS4XpgqM5Cj9JomMzaLylauaGNW6-OhUlA2FZ4ppMIDT42q5iLcgSMiBtwsikEqY2BG0aq8BI1kbxNMONLbAsYQ+NEeoPVSUBoa2Jt0g+W3BNV2h41QP4SHRrQFo7hTBEeyPEANSp3b1rIyxmxLQKqQ3VqLNbW8Htls625dGkkaArUQH62w8htHA6TRlpUosBf4JWjupVPU6SrgV4omVdCrtAYBHQtIEdaYWjVZ9Y1GFKdUmppkMznY92h0E6Dg0V8S1ea4uAWslxbyB0daYdBWoSSiqleiG-5lgoS1+qm1KGj7ehoz4ItsNV2izo2qrXmavaUAdkA2DPHpQysA7OJKpuJ5xb2eMUZ4G8E15c8v+R+BCffiQnRaTVoLGnXDG+B-AaChUwVfmWTVRM8iM23tlGT011T+FjUpiSwHOy86OeF0KMjIsu11sueYUtcsR3raAhG2DAfdjnXiLOVcF+ujttP2F1RjPcECI4CQl8BjtRgFAfYM7G8CBUii9xO-CICOQlYZQiFF4uNBd0LFOo94ioY4kQrM74Jd+U-BNn9Zi6hRxYjXaQAil4q6iva66uPOD2uFUFyMf1nRri6uEeiWqtPbnothx709q3I3AjsM1064ojOgElyPUXgcl2Y40Jbo3Ejos08TwNaXT1rAqsKGA8KcB3pMTd79AZO-ePGQH1d7YANhWuGgEVLDRVxgIW0B2OJIwBOWm8-TYjv+Qw6nyaUIaBiw5w+BB9akSWE13oEwhE0EsWfJzlyAgU4KT62BCWWgAAByLAPAgTCSxDS3cLAqftcIDRdkFZPIv3s72GJu907NBAfon22Fty+CLnvXpimN6pWuWvhMmz5glsz4U2-EkqpH3ZRQFuQVA3nirJz6bCUfUHp2C32yiTiKoM0ioQzKhou2Jsp4o+VgOjiEDze5NpRG0hBB64ZpdA+sXYPOyTsJeBdmQe57a9OdYBoAwCDUikB4Y+zChlyPaLc6EmGEmLQvAq0jjNRo1LA3CAADEoSgmFMEf2sG9hj+omAaqOjuS+wT-BjewJF5TBjQy-SomkOADiSsiUwCctAHyjL8oVQQrrbdsYE8HdYvamPf0AeD07Xg07L-p+JymZ6VDXOwEI4e2KawojpSGvfhV6qOxGoinLUVKy0OYBdDvh-Q5gEMO+GZqJh-oHlGJA2HktrQuoAYCL3sCC9liUveUPVZHCi9LRncgkaqPxGtibhzgbkCIColvDDR81IeSY27MeenO9I0NCuQN6j22o3I-kfFIGGjDjAso+didjfA20lAoHskK6M7HsppSPo5OUGMU7MUqU0Y3EnGPUyk9qh3DYSqyMeMSjybJuIVgGivLKkPeishABtxxlMcK9DIHVktH8s2p9JSVk6QgA1wCuFDLQzgeoKAgPYDQLeeTHkT-GCYwhpgxobimrGfG-wNZAEZTU2L-mayAXUIQ3GYnYpTe544lJuJBFxEdreAASbxi0mfc4wVWAycUBgI9szrOaMIYHLL79AYCDcbCQDZ4n8A07Ck-AeJVIHEpAYP8apJpyJBsgwodFmkCZOwJMp5pHozlNIB5SSpkMUQLriIOHjkiOU-mejTU5lbnKvVeThAEePYnqT2iqcGKfyb7AGwdE5U4azVOWnFG3VYhFVEwDoEo4nkrU7UFyA4Bsc7gDdvOyIBkEwiU0w03AGrFRxvQVcCbPgPQWULOdSqiIv2Nr3MLddzbWI4MXcBfYmyh0PXavzyVwANZhg9zfOyhCjEhQMhpGCC1nZCK4D8xpnH0BLwjQ+udPC6ItJjkpVJlCcswNsDoBAiaYPEH2gJH9qpyntzYJnXkV2XmpnTWUcU7EbRhycK29psciCB3IDQnQJvYmEEDjb+jNT7U7U4RN1MeB8pOgA00aaKlw4+ZmAYiavwu1Xs-TxZZ8iVAkD9BfJ3h8WCGtuU4lHl5SPU8lIfPpnlhzlY6E+cBB5nmGyJ18++d2JWmEk8FpM0kSQuFkNyIcayNQb2pSF94QBfQhuYgJhFBD95y0ZeZcMIwbm38Q6RufyaFNj63hVk5OooALgmQuFj7gRdfyrRAzGWTuGIxZMrAiAWhSQ4dAIJ-4asHFiS6QVTziXfClFxgk9CqWBFOLIRRHCIVQJCW6KlgKJHCAgwuQKsO+Q4MipahvRi4higCYYnt0EtTIWcCy1AHsK4sRE+LewDPNktx5NgDANy1liRJIqIQnAJS+QQ8DstWiWZ8pNNjEYhW-jnMG5trpYVsLn6cV+wuhc+jIF9LP+PE3osHaJJQSgcktTISGhyFDW0HEdnAHhEUjfLecTYLVnqwNZ9W-WfvJ9CiK7JHiDrXc5VeNanwartINlK0W1Y8mIAhUGqF7w3hbwqSiQLI+ikRj3wK2kHG0hDkwBQ5JWIpXhayKyQcjgLrFMaxNYxJ2Essz9S1hWyOv6lOMU4Y0mwTmP3ZFrtplXedbmgTWFEmota+KXQvVTykqLQQ1FTUg8x94b+ncdBWVYqnzam08pDFetMZZ6QwgaDlaxi5TQrgLBVOGWwpZvXbTAhOkGSZgMZYpwkwhGATH1ZNB4biNithAvJs8gKWJ2Gm5Tf9Z9XDE1V2q1gEZvosqrJrVm3TfJYVtzdlGNAETe+BUkbLVdYsUbGpu825oPN5ItLa3gwgn0raGABFGO5E5ERpAJm50kFDc3MOFbCkgwMVtpIVbROPTZKa7MsAn+gYUnmzG5RA3sov0pc3LKVVojMgGIlgByTWy4jMAl2a7K+RHZYIbbhqSePbcwDUjgcfkvXkUeHYmtH9vjR4CReZKcV1rDI1ZbVLXNsiO6HIrhT5pA5qH8NWJscnZLgAOSxoAegG7beDvxgVd+K+45VoLstRLACN+m9Ld1ty3iQsIF8EbdVv+jtztd9Q5SZVOKtObA1nW83Z+Py2O7St42-6Iu0GTJjOG4cfnYHstTFQoEYm4Vi87yrUjNbAs6wsrP4g7mdzF6+NaXLY3BC+gajfFysaa2WbQ1ikTc2dvEnGAx9iaxWq7oc3+rW4Qa6ygpG6Xv8uVzGq8cz5eAMA-Ib3GNHrg2JYA-xD3HdFPnyEwFShMaKoTeVRV4FiSbQjJZjzwPgAbl7wNWCiUJ2yAknSTkFyIcXQ+YZVpssFVwfMQCHUwTHCQ+dFkPMcFDoKVlHau4wt0ZFVM3oD5jQpETEAI0JqLHj7BGHJwVolZRSgqgoHVdfQG7EVAIwZHaoDUFqCGBGExHOYMeMjFQWFMjYXUNAMo+gfP1MDmouEx6AZA2A+gxIBgBFBJyO6hHRD0R7eOIcSOKGpj8VnCerO2P7HEsRxzaWcfiO1OHjsrF494i+AfHmNIqvdyRNOOTgmj2ENo-celNYTRbKW3adicBP4nLjpJ+LAwtWc87e5vthA9VDQOOJ947UJt3dP2S3jY0WzX8kI6RZsrelsQp3iAe7jRAeYUAc2t-05gqHchWh-g8UIiAVyEEMgWrBgeDZyLp4HhwDABvCALHVjmx42TDwOPhH4rQJ1o4keeC8QFQRswrdEYCLEW+z2EHmfFgDxOnSz0nqQBBpwA8kDAR+3KG5BLYRq5SFbGtg2y+NJhrwfssSCzkAkfbN2VifSQBJh2QczpOItiNxE0c7gDAR-STsf37Bn4F5WO9tAmeP7-nCSD0PHebVPAA8VwM4JtiuAHAIQEIcpNi+dGrW6RUrd+OUn0ijOQ6NyER-E7-jQnFGez6EHrk7uwANFb8LQ+wKyc+NLn1j8kDm2As-GsohuVx9bgBWZQlAwrqnJKyyfs5LHIrmwMIX-utOhLQD6OwNaKvZQLREEAaJA7MjYPyrNDwwUg5UL3K0HeYDB9JYYNCW4H5rzYHg-odxPPczDiBEaPeN5R+nNDt18M-EeeuWoECVh76-DMcO9CXDo9EpSiJpPZbGT-x8y4SfSvPBo18tqCSmv63ZrKYVazqyWtzQVrH1ukSnYSTKvWXuxfcmNcREnXLAz9BoL8aCeAm5XLAVuxAGhzKuCQY1zLnNBxs14sruMHK1q5-zVnMA3Ew7gKEsBgAnxZ8LOBJGnezuII46De8cizhDmdom79k9uTyjpAxzrKPLHTzMDshKwH4iFHT2JBaQEYM75KGfEzBBdMwuyH2tjn0hHIKg-r113Q+GcrPVy4z4aB+8GfuusnIbshz64gfXvZ3jrlPNM68CzOq4U7+9OE8UA-vSlcTyR6QnBAnO4QpAMRsc85dCHCmd0rDbbqutukPS6ycEL2VrFtjrc13bcgEn9IMuwsTONF6oorWEVIzYokM1ebfJMfAKBtuODVC6lDQ5QIzpsnO-9MnF22Q4y3aecgSwiTTj+k8Xx+q6Vh3EBzkYN1F3F2PvuRAbibkF2iO6sA3ErLLeLk8oedw1YGAK4gQByeBdngPkt830+njr4In+wMjAYAghAcM0AUJ-0kXmhQr5z2lQiOzdygtQO0DRW89WzrYe+4lTMN81+AMdMsfYCz8x5wZjQ4uwVILqDaCDsT-xZD7qQZ96nOVEvXYZgMSwAErZWm1ImaAwHGgkungFLj0FyWYBMARq77trx2vq8QhGvjwZrwwEfjRb7IONUnBjR6KpfWmXnh4FLG6+9fsRLX5bu1-BCLeuvDXpry18G-Ixhvo3zyK01WwrYxP5ZYlqEcQrIwPQAeLKEqCuVMehYqmeLyoSS+pt7izpVL+zkRgZfrEWX7snJ7y+qSCvfJZz7xJK8MdyvsNSr3cGq-eepYhm3TZN+88rDwfRkjyDV5m9re+vC3trx182pwsJoaP+bwN6G+YARv5VbyON6Y9I+8YU32r7N-W+telvvgFbzj5p-o+CfW3onzt4p+nF-cIOCb6ndOKPA9vAv5ymd4DxPvwQ138T2eHKSAvuPILhJGC6SlBA4i0eLF-HapdJ2VfVMOJbDwBJY6BEY6xFrqGI9Cg9fJv43+6hTeVv2XmH-D9wJ5djV+XlAwV7-C0apefGfMas2K-oSLFFHlgUspL53BSa2UNuK76l92IDwxfvgf382SD8Ujt34vsP0F6wCR+zeie-a6y19-R-VyMR9BSn7BJIes-EEE4pd4RiF+cV-+qWHn56KibnKefhPXId9M52hEHH3IFGbEZGiJ3rC6AJYATCBATTJBgUHFV3ERElPh31cgnoSKvb4tg0VOPc8eceBnnlgZbNF8+cjUPmhImX8C-GIK-1fT8YUh6--jCGjYgo3tsoc511+3zDfpp+Qumwp+-7AlrOEA+rMAfA3hD1L7TisL-uzX1DhB-AqtcoPPjW100JlwWsCwdYHHB1f87DZlxA9vXTUTYdKZb-wGdIAnJ2dFSHMNxzB4A2p04cAobhxTM5nD3yQ9GPAP2Y9MnFNxQD0PQMQv8lVTWAQ9vHV7zQ8q3CVztNm3WV2BMQAFP07cyA+gJT8NXB-wqx5IDnCa5QPOAO-1CscCB0J5PIQAiJNfJ1xwcACNPFn15JGDm9weYOTz5A59E01DcxEIhxkCf8Z1x-9gAGoCmB+TQsRLQMyRui0DdxEwJNMAPIwOFA1YLMXn0zArBQsCCvMGwcCRGOX07xY8Bq2cM98WsCmADzK0FxZquZqHANgDNSHYkeeFsyg8yLGNxmd29CQ270p9AgyUDF9CCWX1G6FkA8DJ9JIkbpWiF4nshn4TMDiJsXHkl+AQQHEQOAA8UgHCDJDLADfhUgpwOR5JPRXRKCQjWKDeBRGVezqDu9cWEKC2gyF2M19AXoI9pWg63XaDrdHDx6CkgsYJ-MOgsIzhY4iKvS6DpgrAFGDUlVoPsgVg14CWCjNa4BGDZgzYPmCdgvYKmDyZDYP6DnyYYPaDTg9IlUDfPIIB+IsgSeEmdyDfYJmh2g1CTWCJYI4KuDMaVCUmDGcWoL+DvfdYN1YjgkYJn1qyFUhRJykdsQyDTAiwK4prdRmxyCnA5EPF1jNTCSW4ASBEI1IbCTENm0DgtEM3Y1QQkLV5T-PoBxD4QkwIpCkdXtlODSQxwPpD4tLEOuAbmabEAND9GAHv9cYAB3sggHUSzDZ7UKODExVJTuGGpEAmh3kCmglUi9xtA9iV3F1AoQDcDquG0l0DMafQIGctDKAPFYB2aCjA95nYM1DwsoJqHFDjg7wJwcJrKYDARqDLBVtMyAQ1x3BpIf8SlCy4GDzjdeHeZzwNJkOUOfMbSEg1rBWiSsmhDCDZElRJgjCXSoU8QU0LPBzpV6TSgzHOUF9CoAc6QYAbSDE0WJSgChgUNBdNkNj1YjamlsIWAQAFByQ4HwAqdZPXntnKYsMV5AAXg3AAFl3+SI5CrDCEYjgHgltFg2AAbSUhR8ZxxIFlwRJqXYyRlykGnBAQMsOMJ3AyHKcLTDJ4adhhtYaJVVTC4TOrxRs0bNglnC0qF1ncQtka+EeZ2cUpl7CsoGuBiC9wiEyhMPzXeybY1FPdg7NmDFdlrAfiWzVnCLoOHj9tc8aAFIV-Rds1l1OzJTn2Bm2be1kVvYIdxadBLH-CDMoELQNfDLUWDD4xVqeaE7g7AJkBqh3Qq0JddgAXUPIdcjAaFE9YIwYHgihMQxXqtqsJqz1AmsbUD1B9WDrCgBusXrDato3HANjc4PUbA8BUw9MMzD03ScJFCw8BMNC9QUJbGEUV-TbC3D7IGX1nD1nMrCPheI1ch38S3DXzjtKXD6y3DZwtay3CH4MtykRewzhCVgs7QvHHxZw86T5DksEd0AcgpV8NdCJQzFmZsTWAD31YpgY+31ZPcCtidDmyKyNJZP7QUGwDhsb0ITdUwjGwrY3WTwWrdM3StlFsIFJyNFsZbK1n9Yoo6qhii9bFQEaEBmP5xjCNgIyMnhEw2ExTDZI+MMngdLDsJ4jeQcQHbhxAOlW3p3pGxyppEYOoVDYSosqMacEkI2Ff1Go460Cs+wC8nyIPQXznEB4yHqKbIrGVqIkBi9WIxeJLbLABOB3iadDqs4Lf0CttopHVjqjRkIeFKiRo-fz5x9gQKF8cIoYKF-gxyVSFhAEYauw8hho8qP0BYcTaLfMdovaKv8K-AK1Wi2ontjrcIFAaIkB+o3qIeiGokaI51EKccPMV-zGcMeiforc13Y+7Je3gM23J738i8o6ADbBxAGGDIA5QRqOhwVAf0WoDzUGGK1sz4eGMRjkYkaMCjXWaKPbcoFcqJpBUootRUA3TWGLnCMAD8xlI8nZp01cII8yPDNO4XVy-sPw8NnHdRgGqA5icWZoVcIYeMiDeCf8TTyE8RPe3nWjpQz9yGc3-EaPJiNUECiuAv3OEDIdkogwApinmHcGjsv7F-1VjHIyWJgA2olWLlitAjWITF3ASQkNN0kAD2axqI0W1oj6I3+CmBMooQATCgY7GIgg4I-ZC-9YHCy3kshgZqwZAF+YQEYiPQ3GHjc8oaDlTDcYo2MaiqPFKKVii1Klnxjyo6mK9joAc6V7h0-ROMVj7AECjiJU4jwEajn6M2wAi-BMaBqdMAmyJf0aYtIEoFko-ON5UpcI7lSl44kaOuMhEKrEHdc4t2O2x9uZBWE11JQ3TCZnCeqQEi4RXaGX8PnTbHEirsG7DUjA7O2x3IFffnnWslIohzWtn4Cl3N4mgDa1pdnKNty8j4AWWTVZ07ezGLjjYkaPeQ9rAEheIm4WoEsAE4aTEHg6JEu2NDEIDVHuifuCU0Kd7rbI2EURGBGBfibQN+OriI3QrC-j7AE6M7pypBmNuMSWcu2wZZDBIn-j-wwBJAAdXE+KeCaY8yABJxwgeFfDPY-2yzj5wosJ3JOdXu0XsinIBOv1SeAaHfDVIpmHRjSmFcL5g1w6aA3DFALcMAhdwqkwnFZqRAy1hLAHxmHtOYkzz7BqwQPDaRPiGHlNEyYgFWTjW45KPVNMLERnSJu4MhTrZrw-eyoSwY+swPYAEyDgkhG0Ve1s1wbV8MYSkOZhMYUF7fXSKcgIoNQQT7om22nZH7dEXrEWAF4BWwqQTbBeBROLmPtRaUWACDtOlHcmlh-JdX32Bko5GEmEPQCEAOA7gX4AG8kXJFyzlNrSqmPjbIgazPiLWC+Mqor4xqPsx9IkCOrDRDNBPBjaEkADskaY6mPfjanY0MsS8Ex21Aj8LfkLMi0sRYmgohAnLz3ENSWXwtIZYwD2GdgPNAPIdjXdFlIj84ciMax7Y1rEdjOsLrHDipneINg88AquDlB5qDUmEpQzDMOZdWiabHn8p4iL1ecASd5xi9gHYEEV9vbBeMGTAQcpAV87SSGBKhY4Rjk75a4YQGRh8hP2S38jxe3TPMFIp+ExctfQ+PLdmXK3zAj9YIhGIhSIdtlaQ6eJoC0NuAC+BAAbGZAFAAhoH7mbwEANAGQA2gEOIJhUUysHLNPjX4CZgJ4RIGQAYg3YGZxSU-0DggwAYkGQAoDZkBAAAVasGEAZQYQBhFyQagFMhzsA10BAPpfFN3QIAVFLZpLAC4BJh4AI0gG5aMFuFgxYkfFLAATQVFKclDEAYEQgWIalJwY2U6QEyByxckHcBLQRUDgBdU-ZlpTbnMgHxTa0ECVRTuXGAAuACYEYGQBonHygJ5BgPUAuAmyfAHxTzCDQlRSaAM1OvBCAFVPQBJ4NUFRS9wCgC1SNuUQAtS9QWlL1A6sFlK0Tk04kBNTqAM1MTS2UsdGdThAV1KQB7HGFSbIMwYkClB3RBNJQBaQY7lRTzMGGBu5c02lIZArQDUBlBYAX4DABrEeiA9Bcgf4HEB3AXNImAJgIAA)

Diff: https://www.diffchecker.com/U6IzLgTQ

Reply:
@minhhoangvn: Hooks are local only - we can't have hooks in repository :(

Reply:
@Saibamen, I think we can use Husky to ensure every commit before creating PR has a consistent code format (https://www.freecodecamp.org/news/how-to-add-commit-hooks-to-git-with-husky-to-automate-code-tasks/). Besides, we can have Github action with format-lint, the sample action flow here has action to check the format-lint https://github.com/zalopay-oss/jmeter-grpc-request/actions/runs/2790320379